### PR TITLE
Add HTML tile

### PR DESCRIPTION
**Pull request by ajayns**
Addresses #292 
The editor works ok but there's some issue with it rendering content once a document is saved.
Possibly due to some configuration issue for the `HTML/View.jsx` part.

Will be working on this, but any pointers on where I should be looking at would be helpful!

### CODE REVIEW

**Comment by ajayns** [comment]
@tisto Yes I missed that. Updated changelog and fixed conflicts

File: src/components/manage/Tiles/HTML/Edit.jsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Editor from 'react-simple-code-editor';
+import { highlight, languages } from 'prismjs/components/prism-core';
+import 'prismjs/components/prism-markup';
+import { Button } from 'semantic-ui-react';
+
+import './styles.css';
```
Comment (sneridagh) [inline src/components/manage/Tiles/HTML/Edit.jsx]: @ajayns Since we are doing all using the SemanticUI engine, we only insert HTML in the Pastanaga theme, that later might be overridable by integrators in Volto projects using `create-volto-app`.

Please place it in there (site.overrides is fine for now, we have to tidy up that file anyways)

File: src/components/manage/Tiles/Text/Edit.jsx [hunk 2]
```diff
@@ -273,7 +273,7 @@ export default class Edit extends Component {
           this.state.customTilesOpened && (
             <div className="add-tile toolbar">
               {tiles.customTiles.map(tile => (
-                <Button.Group>
+                <Button.Group key={tile.title}>
```
Comment (sneridagh) [inline src/components/manage/Tiles/Text/Edit.jsx]: This is nice, but maybe it belongs to another PR, for separation of concerns.

File: styleguide/index.html [hunk 3]
```diff
@@ -0,0 +1 @@
+<!DOCTYPE html><html><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width, initial-scale=1.0"><meta http-equiv="X-UA-Compatible" content="ie=edge"><title>Volto Style Guide</title></head><body><div id="rsg-root"></div><script src="build/bundle.0ac92aab.js"></script></body></html>
```
Comment (sneridagh) [inline styleguide/index.html]: Where this come from?

@ajayns @nileshgulia1 ?

File: styleguide/static/media/Logo.912d8f6d.svg [hunk 4]
```diff
@@ -0,0 +1,83 @@
+
```
Comment (sneridagh) [inline styleguide/static/media/Logo.912d8f6d.svg]: Where this come from?

@ajayns @nileshgulia1 ?

File: styleguide/static/media/pastanaga.08f24832.svg [hunk 5]
```diff
@@ -0,0 +1,83 @@
+
```
Comment (sneridagh) [inline styleguide/static/media/pastanaga.08f24832.svg]: Where this come from?

@ajayns @nileshgulia1 ?

**Comment by sneridagh** [comment]
@ajayns @nileshgulia1 I also fiddled around for prettyfy the output, but I'm afraid we would need an external trigger for it. We currently have no handler that executes something on tile blur, that might be handy... /cc @robgietema 

FYI I used pretty npm package.

File: src/components/manage/Tiles/HTML/Edit.jsx (hunk 1 above)
Comment (nileshgulia1) [reply src/components/manage/Tiles/HTML/Edit.jsx]: Added :+1

File: src/components/manage/Tiles/Text/Edit.jsx (hunk 2 above)
Comment (nileshgulia1) [reply src/components/manage/Tiles/Text/Edit.jsx]: gotcha.

File: styleguide/index.html (hunk 3 above)
Comment (nileshgulia1) [reply styleguide/index.html]: Sorry. It did came along with styleguidist. I'll remove it. :+1:

File: styleguide/static/media/pastanaga.08f24832.svg (hunk 5 above)
Comment (nileshgulia1) [reply styleguide/static/media/pastanaga.08f24832.svg]: Same. removing...

**Comment by sneridagh** [comment]
Now I think it's ready for prime time :)

/cc @nileshgulia1 @ajayns

**Comment by tisto** [comment]
@ajayns @sneridagh functionality looks great!!! Do we currently filter the input in any way or is that something that we have to take care of in the future?

**Comment by sneridagh** [comment]
@tisto I already thought about that, we can surely look for it in the future, but for now it's fine if we trust our editors, of course!
